com.unity.netcode.adapter.utp/Runtime/UTPTransport.cs

```diff
@@ -0,0 +1,835 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+using UTPNetworkEvent = Unity.Netcode.NetworkEvent;
```

Comment:
Is there a reason for renaming the NGO `NetworkEvent` as `UTPNetworkEvent`? I understand why we're renaming in the first place (to avoid a namespace conflict), but I would expect the `NetworkEvent` from UTP to be named `UTPNetworkEvent`, not the NGO one.

Reply:
That was rider renaming :P the main issue because conflicting Types since UTP and NGO both have a NetworkEvent type